src/scripts/list-users.test.ts

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable functional/no-return-void */
+/* eslint-disable functional/no-expression-statement */
+/* eslint-disable functional/functional-parameters */
+import * as TE from 'fp-ts/TaskEither';
+import * as O from 'fp-ts/Option';
+import { UserService } from './services/user-service';
+import { users, usersInGroup } from './list-users';
+import { GroupService } from './services/group-service';
+
+import * as test from './services/testing-service';
+
+describe('Listing users', () => {
```

Comment:
Let's have separate describe blocks for listing all users and listing users by group.

Reply:
Blocks seperated